lab-ben/README.md

```diff
@@ -0,0 +1,13 @@
+# Telnet ChatRoom
```

Comment:
Good README, no points lost, but note the directions:

```
Write a paragraph about your project
Write docs on how to get the project running
Write docs on how to connect to the server
```
You're missing the paragraph to describe your project, but I won't dock points for it this time.